File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
        char[] s1 = s.toCharArray();
        char[] t1 = t.toCharArray();
        //given s and t has the same length
        int n = s1.length, ans = 0;
        
        int[] counter = new int[26];
        for (int i = 0; i < n; i++) {
            counter[s1[i] - 'a']++;
            counter[t1[i] - 'a']--;
        }
        
        for (int count : counter) {
            if (count > 0) ans += count;
        }
        
        return ans;
    }
}

File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.py
class Solution(object):
    def minSteps(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: int
        """
        count, ans = Counter(s), 0
        for c in t:
            if count[c] > 0:
                count[c] -= 1
            else:
                ans += 1
        return ans;